typedoc.ts

```diff
@@ -1,3 +1,4 @@
+export * from './src/element';
```

Comment:
somehow I missed this. Would you mind to explain this?

Reply:
> I have created `./typedoc.ts` to put all the elements in a single module in order to facilitate the review activities during the migration. With this approach all the classes, interfaces & type aliases are accesible in a single page under `exports`.

from #874 this file includes the files to document with typedoc

Reply:
but docco is still used. Should not it be deleted then?